main.nf

```diff
@@ -233,6 +244,11 @@ workflow {
                 )
             }
         }
+    if ('coverage' in params.algorithm) {
```

Comment:
**suggestion**: We may want a different name than `coverage` since additional coverage assessment tools may be added to the pipeline in the future